signer/src/transaction_coordinator.rs

```diff
@@ -553,6 +619,15 @@ where
         PublicKey::from_private_key(&self.private_key)
     }
 
+    /// This function provides a deterministic 32-byte identifier for the
+    /// signer. This should probably deterministically too.
```

Comment:
"This should probably deterministically too" ... ? :laughing:

Reply:
lol will update.